server/sources/ElectionManager/ServerElectionManager.h

```diff
@@ -0,0 +1,64 @@
+//
+// Created by Douglas Gehring on 20/11/20.
+//
+
+#ifndef SISOP2_T1_SERVERELECTIONMANAGER_H
+#define SISOP2_T1_SERVERELECTIONMANAGER_H
+
+
+#include "Message/Message.h"
+#include "ServerDefinitions.h"
+#include "SharedDefinitions.h"
+
+class ServerCommunicationManager;
+
+
+/** Each time a server start a new election
+ * it needs to call setupElection() and then
+ * send a message with its id to the current
+ * connected server using the electionConnnection
+ * socket
+ */
```

Comment:
Se o server vai dar start na election então melhor chamar de `startElection()`, não?
```suggestion
/** Each time a server start a new election
 * it needs to call startElection() and then
 * send a message with its id to the current
 * connected server using the electionConnnection
 * socket
 */
```

Reply:
Acho que a ideia é que ele tem que configurar várias coisas, por isso `setup`, `start` seria quem chama o `setup` talvez =P
Mas acho que tanto faz